store/src/test/java/com/dropbox/android/external/store4/impl/FlowStoreTest.kt

```diff
@@ -46,8 +46,10 @@ import kotlinx.coroutines.test.runBlockingTest
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.junit.runners.JUnit4
+import kotlin.time.ExperimentalTime
```

Comment:
Ditto. We shouldn't need this anywhere in the tests unless it's configuring the `MemoryPolicy` via the builder.